modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,77 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+:_content-type: PROCEDURE
+[id="overriding-default-node-ip-selection-logic_{context}"]
+= Optional: Overriding the default node IP selection logic 
+
+You can create a hint file that includes the `NODEIP_HINT` variable to override the default IP selection logic. This allows users to select a specific node IP address from the subnet `NODEIP_HINT` variable and specify which IP address is used. For example, a node that has two interfaces, `eth0` with the subnet range `10.0.0.10/24`, and `eth1` with the subnet range `192.0.2.5/25`, will by default route traffic to `eth0` and use the `10.0.0.10` address. `NODEIP_HINT` can be configured to point at `192.0.2.1` so that the correct subnet, `192.0.2.5`, is used. 
```

Comment:
...
This allows users to select a specific node IP address by specifying a reachable IP from the desired interface's subnet.
The `NODEIP_HINT` is normally the API VIP.  The IP selection logic will determine which subnet the `NODEIP_HINT` belongs to and from that determine the interface in that subnet and select an IP on that interface.  

For example, a node that has two interfaces, `eth0` with the subnet range `10.0.0.10/24`, and `eth1` with the subnet range `192.0.2.5/24`, and with a default route via the `10.0.0.10` source address on `eth0`, will by default use the `10.0.0.10` address for the node.  
`NODEIP_HINT` can be configured to point at an IP of `192.0.2.1` (presumably the API VIP on `192.0.2.0/24`) so that the correct subnet is selected and the `192.0.2.5` address is used on `eth1`.